src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.UI/Views/SecondPage.xaml.cs

```diff
@@ -5,8 +5,29 @@ namespace MyExtensionsApp.Views;
 public sealed partial class SecondPage : Page
 {
 	public SecondPage()
-    {
-        this.InitializeComponent();
-    }
+	{
+//+:cnd:noEmit
+#if markup
+#if(reactive)
```

Comment:
Finally seems that we should keep the `reactive` flag :)

Reply:
So I assume that here we should have :
```suggestion
#if(mvux)
```

Reply:
no... reactive never went away... it just changed to a computed parameter so this isn't something that would change in the template